src/app/components/header/Header.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component, Fragment } from 'react';
+import styled from 'styled-components';
+
+const BBCNewsBranding = styled.div`
+  background-color: #bb1919;
+  height: 58px;
+  width: 100%;
+`;
+
+const BBCNewsBrandingLogo = styled.div`
+  margin: 0 62px 0 62px;
+  padding: 10px 16px 10px 16px;
+`;
+
+const BBCNewsNavigation = styled.div`
+  width: 100%;
+`;
+
+const BBCNewsNavigationBrand = styled.a`
+  font-family: ReithSans, Arial, Helvetica, freesans, sans-serif;
+  font-size: 2.4em;
+  color: #fff;
+  text-transform: uppercase;
+`;
+
+class Header extends Component {
+  state = {
+    title: 'BBC News',
+  };
+
+  render() {
+    const { title } = this.state;
+
+    return (
+      <Fragment>
+        <BBCNewsBranding>
```

Comment:
On the outermost element in this component, we should use the semantic element `header` instead of a `div`. There should also be these attributes on the header element `role="banner" aria-label="BBC News"`. These are used by assistive technology to determine that this is a banner with the label BBC News.